docs/src/getting-started/custom_dataset_conf.rst

```diff
@@ -116,21 +117,14 @@ Each gradient section (like ``forces`` or ``stress``) has similar parameters:
 :param file_format: The file format, guessed from the suffix if not provided.
 :param key: The key for reading from the file.
 
-Sections set to ``true`` or ``on`` automatically expand with default parameters.
-
-.. _energy-section:
-
-Energy Section
-^^^^^^^^^^^^^^
-The ``energy`` section is mandatory for MD simulations, with forces and stresses enabled
-by default.
-
-- A warning is raised if requisite data is missing, but training proceeds without them.
-- Setting a ``virial`` section automatically disables the ``stress`` section in the
```

Comment:
did you remove this thing on the ``virial`` because it is not true anymore?

Reply:
No I moved this to the general section because it does not only apply for the `energy` section but we do this for all sections. Something we might want to change but this is how it is implemented for now.